cypress/support/step-definitions/menu-steps.js

```diff
@@ -10,6 +10,7 @@ import {
   menuComponent,
   submenuItem,
 } from "../../locators/menu";
+import { SEARCH_COMPONENT, CROSS_ICON } from "../../locators/search/locators";
```

Comment:
we should create a new functions in `index.js` file or use already existing one from `search`.